scripts/jenkins-report.sh

```diff
@@ -1,17 +1,39 @@
 #!/usr/bin/env bash
 
+# Note: this script is in a temporary state. In an effort to get the
+# platform off of coveralls and onto codecov, we will be temporarily
+# using both.
+
 # This script is used by the edx-platform-unit-coverage jenkins job.
 
+if [ $# -eq 2 ]; then
```

Comment:
I'm decent at bash, but I have to admit; I'm not sure what `$#` does. Does it count the number of parameters?